hooks/pre/10-reset_feature.rb

```diff
@@ -57,4 +57,12 @@ def reset_pulp
   Kafo::Helpers.execute(commands)
 end
 
-reset if app_value(:reset) && !app_value(:noop)
+if app_value(:reset) && !app_value(:noop)
+  response = ask('Are you sure you want to continue? This will drop the databases, reset all configurations that you have made and bring the Katello server back to a fresh install.[y/n]')
```

Comment:
Can we drop the 'Katello' in the middle there? I think server should be sufficient.